### ISSUE TITLE
Logs for unreadable/non-existent certificates

### ISSUE THREAD

**Issue by STotev** [composer/ca-bundle#65]
Hello,

These days I have been dealing with issues around Composer which ended up being an issue with the certificates making Composer unable to download packages.

Is it possible to add more logs, in verbose mode, for giving more information whether the selected `cafile` exists and is readable? Or to be more precise, to tell us that the file does not exist or it can't be read.
Since it already gives information whether the found certificate is valid, it feels incomplete when there is no information when it fails to find or read the file. It would be really helpful while debugging such issues.

I didn't know where Composer is looking for certificates so I had to check how exactly Composer works to get to the point that it uses `ca-bundle` package where I could see the actual paths that it's checking, in order to get to a conclusion.

Thank you.

**Comment by Seldaek** [comment]
Ok so it would be somthing like adding logs if is_readable fails in https://github.com/composer/ca-bundle/blob/9fdb22c2e97a614657716178093cd1da90a64aa8/src/CaBundle.php#L340-L352 or perhaps also if the glob() fails for dirs. Would that be enough?

**Comment by STotev** [comment]
@Seldaek yes, I think this should be enough. I could create a PR but wasn't sure what the decision for that might be. Should it be just debug logs or should it throw an exception? Since unreadable or inexistent certificates are essentially invalid, should that stop composer from continuing? Although this is probably a question for the Composer itself.

**Comment by Seldaek** [comment]
I think just debug logs so if you run composer with `-vvv` you  get a chance to see those warnings. I see no reason to abort due to an unreadable one if an alternative usable CA file can be found.
